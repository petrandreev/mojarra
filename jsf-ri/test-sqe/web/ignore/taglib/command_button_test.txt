      <form id="form01" method="post" action="/jsf-systest/faces/taglib/command_button_test.jsp;jsessionid=F7DE89B48158F37AED9E48FA5D2E93F5">
