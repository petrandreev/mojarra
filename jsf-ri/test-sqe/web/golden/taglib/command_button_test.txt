


<html>
<head>
    
    
    <title>command_button_test.jsp</title>
</head>
<body>
    
    
    
      <form id="form01" name="form01" method="post" action="/jsf-systest/faces/taglib/command_button_test.jsp;jsessionid=F7DE89B48158F37AED9E48FA5D2E93F5">

        <input id="form01:button01" type="submit" name="form01:button01" value="My Label" />
        <input id="form01:button02" type="reset" name="form01:button02" value="This is a String property" />
        <input id="form01:button03" type="submit" name="form01:button03" value="RES-BUNDLE KEY" />
        <input id="form01:button04" type="image" src="duke.gif" name="form01:button04" />
        <input id="form01:button05" type="image" src="resbundle_image.gif" name="form01:button05" />
      <input type="hidden" name="form01" value="form01" /></form>
    
</body>
</html>

